packages/httpjson/data_stream/generic/manifest.yml

```
@@ -0,0 +1,228 @@
+title: Custom HTTPJSON Input
+type: logs
+streams:
+  - input: httpjson
+    description: Collect custom data from REST API's
+    template_path: httpjson.yml.hbs
+    title: Custom HTTPJSON Input
+    vars:
+      - name: data_stream.dataset
+        type: text
+        title: Dataset name
+        description: |
+          Dataset to write data to. Changing the dataset will send the data to a different index. You can't use `-` in the name of a dataset and only valid characters for [Elasticsearch index names](https://www.elastic.co/guide/en/elasticsearch/reference/current/docs-index_.html).
+        default: httpjson.generic
+        required: true
+        show_user: true
+      - name: request_url
+        type: text
+        title: Request URL
+        description: i.e. scheme://host:port/path
+        show_user: true
+        required: true
+        default: https://server.example.com:8089/api
+      - name: request_interval
+        type: text
+        title: Request Interval
+        description: How often the API is polled, supports seconds, minutes and hours.
+        show_user: true
+        required: true
+        default: 1m
+      - name: request_method
+        type: text
+        title: Request HTTP Method
+        description: Supports either GET or POST
+        show_user: true
+        required: true
+        default: GET
+      - name: username
+        type: text
+        title: Basic Auth Username
+        show_user: true
+        required: false
+        description: The username to be used with Basic Auth headers
+      - name: password
+        type: password
+        title: Basic Auth Password
+        show_user: true
+        required: false
+        description: The password to be used with Basic Auth headers
+      - name: oauth_id
+        type: text
+        title: Oauth2 Client ID
+        description: Client ID used for Oauth2 authentication
+        show_user: true
+        required: false
+      - name: oauth_secret
+        type: password
+        title: Oauth2 Client Secret
+        description: Client secret used for Oauth2 authentication
+        show_user: true
+        required: false
+      - name: oauth_token_url
+        type: text
+        title: Oauth2 Token URL
+        description: The URL endpoint that will be used to generate the tokens during the oauth2 flow. It is required if no provider is specified.
```

As there is no `provider` field, is unclear what this refers to. 

If it refers to `provider` property in `oauth_custom` I would phrase this differently, for example:
```suggestion
        description: The URL endpoint that will be used to generate the tokens during the oauth2 flow. It is required if no oauth_custom variable is set or provider is not specified in oauth_custom variable.
```